packages/plugin-selenium-driver/src/plugin/index.ts

```diff
@@ -493,7 +468,7 @@ export class SeleniumPlugin implements IBrowserProxyPlugin {
         const client = this.getBrowserClient(applicant);
```

Comment:
Rename this into `selector` and make it into 
`const selector = await this.getBrowserClient(applicant).$(xpath)`.